cmd/amp-cluster/bootstrap.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"log"
+)
+
+var bootstrapCmd = &cobra.Command{
+	Use:     "bootstrap",
+	Short:   "Bootstraps the node",
+	Long:    `Installs Docker, setups Swarm, run the AMP agent`,
+	Aliases: []string{"bs"},
+	Run: func(cmd *cobra.Command, args []string) {
+		client.bootstrap(cmd, args)
+	},
+}
+
+// Flag variables
+var SwarmManagerHost string
+var SwarmManagerPort string
+var SwarmJoinToken string
+var AmpImageTag string
+var CreateSwarm bool
+var Manager bool
```

Comment:
These vars should have their first letter lowercase concidering they are only local